src/test/java/com/michalsydoryk/app/boardchecker/Board2DCheckerTest.java

```diff
@@ -0,0 +1,13 @@
+package com.michalsydoryk.app.boardchecker;
+
+import com.michalsydoryk.app.board.Board2D;
+import com.michalsydoryk.app.coordinates.Coordinates2D;
+import com.michalsydoryk.app.sign.Sign;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+@Test
+public class Board2DCheckerTest {
+
```

Comment:
Why empty test?

Reply:
not finished yet

Reply:
So maybe mark it as todo / fixme or by comment